src/subscriber.go

```diff
@@ -104,3 +112,44 @@ func (s *Subscriber) Unsubscribe() {
 
 	s.logger.Info("unsubscribed")
 }
+
+func (s *Subscriber) writeRTCP() {
+	if s.Track.Kind() != webrtc.RTPCodecTypeVideo {
+		return
+	}
+
+	for {
+		packets, _, err := s.sender.ReadRTCP()
+		if err != nil {
+			if errors.Is(err, io.ErrClosedPipe) || errors.Is(err, io.EOF) {
+				return
+			}
+
+			s.logger.WithError(err).Warn("failed to read RTCP on track")
+		}
+
+		packetsToForward := []rtcp.Packet{}
+		readSSRC := uint32(s.publisher.Track.SSRC())
+
+		for _, packet := range packets {
+			switch typedPacket := packet.(type) {
+			// We mung the packets here, so that the SSRCs match what the
+			// receiver expects
+			case *rtcp.PictureLossIndication:
+				typedPacket.SenderSSRC = s.ssrc
```

Comment:
Hmm, not sure, I don't think it worked without this :thinking:

Reply:
Do you by chance know what the other SSRC was equal to? - I'm just wondering so that we know which SSRC refer to which values and if it does not work, we should at least understand why 🙂